packages/auth/amplify_auth_cognito_dart/lib/src/state/machines/sign_in_state_machine.dart

```diff
@@ -149,6 +156,7 @@ class SignInStateMachine extends StateMachine<SignInEvent, SignInState> {
           return worker;
         }
         worker = ConfirmDeviceWorker.create();
+        // TODO(dnys1): Log
```

Comment:
Should these be switched with logs or removed?  Would we just need to update `print.dart -> safePrint` here?